src/constants/skills.js

```diff
@@ -127,11 +128,21 @@ const icons = {
     blender: 'https://download.blender.org/branding/community/blender_community_badge_white.svg',
     babel: 'https://www.vectorlogo.zone/logos/babeljs/babeljs-icon.svg',
     sketch: 'https://www.vectorlogo.zone/logos/sketchapp/sketchapp-icon.svg',
+    flask: 'https://www.vectorlogo.zone/logos/pocoo_flask/pocoo_flask-ar21.svg',
```

Comment:
Please use this instead: https://www.vectorlogo.zone/logos/pocoo_flask/pocoo_flask-icon.svg
Do not use icons with text.